modules/KalturaSupport/components/related/related.js

```diff
@@ -266,12 +266,11 @@
 			return this.getEntriesFromPlaylistId( '_KDP_CTXPL', callback, true);
 		},
 		isValidResult: function( data ){
-			// Check if we got error
+			// Check if we got error and if entry list is empty disable the button
 			if( !data || data.length === 0	||( data.code && data.message )	){
 				var errMsg = data.message ? data.message : 'No related items were found.';
 				this.log('Error getting related items: ' + errMsg );
-				this.updateTooltip(gM('mwe-embedplayer-related-errMsg'));
-				this.onDisable();
+				this.getBtn().hide();
```

Comment:
So with a playlist the related is disabled for the entire list of entries and will not come back if the condition is met.